<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {background-color: black; color: white}
QPushButton{color: black; }
QWidget::QLabel{font: 24 px; color: white; padding: 2 px; alignement: center}
QTabBar::tab {color:white; background: transparent; 
border-top: 2px solid white; 
border-left: 2px solid white; 
border-right: 2px solid white; 
text-align: center;
padding: 10px; font: 28px; width: 136px
}
QTabWidget::pane { /* The tab widget frame */
    border-top: 0px solid black;
}
</string>
   </property>
   <widget class="QTabWidget" name="tabMain">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>800</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabUser">
     <attribute name="icon">
      <iconset resource="ressources.qrc">
       <normaloff>:/images/two-way-arrows.png</normaloff>:/images/two-way-arrows.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QPushButton" name="btnOn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>201</width>
        <height>211</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="ressources.qrc">
        <normaloff>:/images/left-arrow-button.png</normaloff>
        <normalon>:/images/left-arrow-button (1).png</normalon>:/images/left-arrow-button.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOff">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>360</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="ressources.qrc">
        <normaloff>:/images/right-arrow-button.png</normaloff>
        <normalon>:/images/right-arrow-button.png</normalon>:/images/right-arrow-button.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabLight">
     <attribute name="icon">
      <iconset resource="ressources.qrc">
       <normaloff>:/images/idea (1).png</normaloff>:/images/idea (1).png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QPushButton" name="btnL1">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="ressources.qrc">
        <normaloff>:/images/idea (1).png</normaloff>:/images/idea (1).png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnL2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>420</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="ressources.qrc">
        <normaloff>:/images/idea (1).png</normaloff>
        <normalon>:/images/BOUTON - SALLE DE BAIN (JOUR).png</normalon>:/images/idea (1).png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>360</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Salle de bain</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Cuisine</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAdmin">
     <property name="styleSheet">
      <string notr="true">QPushButton { background-color: white }</string>
     </property>
     <attribute name="icon">
      <iconset resource="ressources.qrc">
       <normaloff>:/images/wrench.png</normaloff>:/images/wrench.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QPushButton" name="btnQuit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>300</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdWaterCons">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>gallons</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
